lib/node_modules/@stdlib/stats/base/dsmeanwd/docs/repl.txt

```diff
@@ -35,28 +35,25 @@
     > {{alias}}( x.length, x, 1 )
     ~0.3333
 
-    // Using `N` and `stride` parameters:
+    // Using `N` and stride parameters:
     > x = new {{alias:@stdlib/array/float32}}( [ -2.0, 1.0, 1.0, -5.0, 2.0, -1.0 ] );
-    > var N = {{alias:@stdlib/math/base/special/floor}}( x.length / 2 );
-    > var stride = 2;
-    > {{alias}}( N, x, stride )
-    ~0.3333
+    > {{alias}}( 4, x, 2 )
+    NaN
```

Comment:
Incorrect. Pause to consider whether returning `NaN` when we originally returned `0.33` makes sense?